assets/src/edit-story/components/fontPicker/fontPicker.js

```diff
@@ -0,0 +1,144 @@
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as DropDownIcon } from '../../icons/dropdown.svg';
+import Popup from '../popup';
+import FontPickerContainer from './pickerContainer';
+
+const DEFAULT_WIDTH = 240;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  color: ${({ theme }) => theme.colors.fg.v0};
+  font-family: ${({ theme }) => theme.fonts.body1.font};
+`;
+
+const FontPickerSelect = styled.div.attrs({ role: 'button', tabIndex: '0' })`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  flex-grow: 1;
+  background-color: ${({ theme, lightMode }) =>
+    lightMode ? rgba(theme.colors.fg.v1, 0.1) : rgba(theme.colors.bg.v0, 0.3)};
+  border-radius: 4px;
+  padding: 2px 0 2px 6px;
+  cursor: pointer;
+
+  ${({ disabled }) =>
+    disabled &&
+    `
+      pointer-events: none;
+      opacity: 0.3;
+    `}
+
+  svg {
+    width: 28px;
+    height: 28px;
+    color: ${({ theme, lightMode }) =>
+      lightMode ? theme.colors.fg.v1 : rgba(theme.colors.fg.v1, 0.3)};
+  }
+`;
+
+const FontPickerTitle = styled.span`
+  user-select: none;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+`;
+
+function FontPicker({ onChange, lightMode = false, placeholder, value }) {
+  const selectRef = useRef();
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleOptions = useCallback(() => {
+    setIsOpen(false);
+  }, []);
+
+  const handleCurrentValue = useCallback(
+    (option) => {
+      if (onChange) {
+        onChange(option);
+      }
+      setIsOpen(false);
+      selectRef.current.focus();
+    },
+    [onChange]
+  );
+
+  const handleSelectClick = (e) => {
+    e.preventDefault();
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <Container>
+      <FontPickerSelect
+        onClick={handleSelectClick}
+        aria-pressed={isOpen}
+        aria-haspopup={true}
+        aria-expanded={isOpen}
+        ref={selectRef}
+        lightMode={lightMode}
+      >
+        <FontPickerTitle>{value || placeholder}</FontPickerTitle>
+        <DropDownIcon />
+      </FontPickerSelect>
+      <Popup anchor={selectRef} isOpen={isOpen} width={DEFAULT_WIDTH}>
+        <FontPickerContainer
+          handleCurrentValue={handleCurrentValue}
+          toggleOptions={toggleOptions}
+        />
+      </Popup>
+    </Container>
+  );
+}
+
+FontPicker.propTypes = {
+  value: PropTypes.any,
+  onChange: PropTypes.func.isRequired,
+  lightMode: PropTypes.bool,
+  placeholder: PropTypes.string,
+};
+
+FontPicker.defaultProps = {
+  onChange: () => {},
```

Comment:
I would not recommend set a default value for this required property. Also, will be useful to add default values for others not required fields.

Reply:
Added default props.